docs/user-manual/en/broker-balancers.md

```diff
@@ -0,0 +1,156 @@
+# Broker Balancers
+Apache ActiveMQ Artemis broker balancers allow incoming client connections to be distributed across multiple [target brokers](target-brokers).
+The target brokers are grouped in [pools](#pools) and the broker balancers use a [target key](#target-key)
+to select a target broker from a pool of brokers according to a [policy](#policies).
+
+## Target Broker
+Target broker is a broker that can accept incoming client connections and is local or remote.
+The local target is a special target that represents the same broker hosting the broker balancer.
+The remote target is another broker reachable by the broker that hosts the broker balancer.
+
+## Target Key
+The broker balancer uses a target key to select a target broker.
+It is a string retrieved from an incoming client connections, the supported values are:
+* `SNI_HOST` is the hostname indicated by the client in the SNI extension of the TLS protocol;
+* `SOURCE_IP` is the source IP address of the client;
+* `USER_NAME` is the username indicated by the client.
+
+## Pools
+The pool is a group of brokers and checks periodically their state.
+Including the local target broker in the pool allows broker hosting the broker balancer to accept incoming client connections. 
+By default, a pool doesn't include the local target broker, to include it the `local-target-enabled` parameter must be `true`.
+A pool becomes ready when the minimum number of ready targets defined by the `quorum-size` parameter is reached.
```

Comment:
is it just quorum (the size is implicit, as in quorum is the minimum number of members that make the group valid.

Reply:
I used the `quorum-size` name just to be consistent with other parameter names, see replicatedPolicyType and replicaPolicyType.